src/app.md

```diff
@@ -367,4 +365,4 @@ fn main() {
 [`WalkDir::max_depth`]: https://docs.rs/walkdir/*/walkdir/struct.WalkDir.html#method.max_depth
 [`WalkDir::filter_entry`]: https://docs.rs/walkdir/*/walkdir/struct.WalkDir.html#method.filter_entry
```

Comment:
should be https://docs.rs/walkdir/*/walkdir/trait.WalkDirIterator.html#method.filter_entry
instead of https://docs.rs/walkdir/*/walkdir/struct.WalkDir.html#method.filter_entry

sorry to not have caught it earlier